rir/src/interpreter/interp.cpp

```diff
@@ -2652,9 +2674,29 @@ SEXP evalRirCode(Code* c, Context* ctx, SEXP* env,
     }
 
 eval_done:
+    while (!synthesizeFrames.empty()) {
```

Comment:
How do we get here from the deopt_ instruction? I thought this would be done immediately at the deopt point. Cannot there be some call in between the deopt and here that would inspect the missing frames?

Reply:
yes, but doing it eagerly is hard, so that was my first approach, and it does not seem to break immediately... but maybe I should try harder to create them eagerly